docs/modules/usage/runtimes.md

```diff
@@ -0,0 +1,77 @@
+# Runtime Configuration
+
+A Runtime is an environment where the OpenHands agent can edit files and run
+commands.
+
+By default, OpenHands uses a Docker-based runtime, running on your local computer.
+But this can be tricky to set up and manage, so we also offer support for "remote" runtimes.
+In these cases, you'll generally need to set up an account with a runtime provider
+and get an API key to use their runtime.
```

Comment:
This reads a bit bad to me. There are advantages with running locally, such as privacy and some control over one's files. Well, from the perspective of the runtime 😅

I don't have great alternatives (idk how much detail do we want to document here?). I would suggest to move info about remote in the section about remote.

Reply:
Thanks for the feedback! I tried to be more explicit about the tradeoffs here